MODULE.bazel

```
@@ -54,3 +54,11 @@ use_repo(python, "pythons_hub")
 
 # This call registers the Python toolchains.
 register_toolchains("@pythons_hub//:all")
+
+# ===== DEV ONLY SETUP =====
+docs_pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
```

nit: should we use something like this?
```suggestion
docs_pip = use_extension("//python/extensions:pip.bzl", "pip")
```

---

Done